userbot/events.py

```diff
@@ -15,11 +15,16 @@
 
 from telethon import events
 
-from userbot import bot, BOTLOG_CHATID
+from userbot import bot, BOTLOG_CHATID, LOAD_PLUG
 
 
 def register(**args):
     """ Register a new event. """
+    import inspect
+    stack = inspect.stack()
```

Comment:
`inspect.stack()` is a Cython implementation detail. Don't use it.

Reply:
Then how must i get the modules name?